src/basic/process-util.c

```diff
@@ -1623,6 +1624,45 @@ int get_process_threads(pid_t pid) {
         return n;
 }
 
+int clone_vm_vfork(int (*callback)(void *), int flags, void *userdata) {
+        static size_t stack_size = 0;
+        char *stack;
+        int r;
+
+        assert(callback);
```

Comment:
maybe add an assert that checks that CLONE_PARENT_SETTID, CLONE_CHILD_SETTID, CLONE_SETTLS are not included in the flags param, since you don't accept them and set them. (and maybe there are other flags to refuse?)

Reply:
I'll add those 3, not sure if there are more, if anybody has suggestions please shout